pycqed/analysis/fitting_models.py

```diff
@@ -112,22 +117,179 @@ def Qubit_dac_to_freq(dac_voltage, f_max,
 
     # using E_c as fit parameter is numerically not stable (changing E_c does only
     # slightly change the function. Set E_c to zero
-    E_c = 0
 
     if V_per_phi0 is None and dac_flux_coefficient is None:
         raise ValueError('Please specify "V_per_phi0".')
+    if dac_sweet_spot is None and phi_park is None:
+        raise ValueError('Please specify "phi_park".')
+    elif dac_sweet_spot is not None and phi_park is not None:
+        raise ValueError('"phi_park" and "dac_sweet_spot" cannot '
+                         'be used simultaneously.')
 
     if dac_flux_coefficient is not None:
-        logging.warning('"dac_flux_coefficient" deprecated. Please use the '
+        log.warning('"dac_flux_coefficient" deprecated. Please use the '
                         'physically meaningful "V_per_phi0" instead.')
         V_per_phi0 = np.pi / dac_flux_coefficient
 
+    if phi_park is not None:
+        dac_sweet_spot = phi_park * V_per_phi0
+
     qubit_freq = (f_max + E_c) * (
             asymmetry ** 2 + (1 - asymmetry ** 2) *
             np.cos(np.pi / V_per_phi0 *
                    (dac_voltage - dac_sweet_spot)) ** 2) ** 0.25 - E_c
     return qubit_freq
 
+def Qubit_dac_to_freq_precise(dac_voltage, Ej_max, E_c, asymmetry,
```

Comment:
Naming is far from ideal and I still don't understand why we have capitalized function names in this module... will have to be cleaned up at some point. But ok, not critical for now.